package.json

```diff
@@ -57,6 +66,7 @@
     "jest": "^26.4.2",
     "lint-staged": "^10.2.13",
     "mock-fs": "^4.13.0",
+    "nodemon": "^2.0.4",
```

Comment:
Where is this used?

Reply:
I used this while testing instead of ts-node at one point, I'll remove it as well